packages/node/src/anr/index.ts

```diff
@@ -86,62 +104,54 @@ function startInspector(startPort: number = 9229): string | undefined {
   return inspectorUrl;
 }
 
-function startChildProcess(options: Options): void {
+function startAnrWorker(options: Options): void {
+  const { Worker } = getWorkerThreads();
+
   function log(message: string, ...args: unknown[]): void {
     logger.log(`[ANR] ${message}`, ...args);
   }
 
   const hub = getCurrentHub();
 
   try {
-    const env = { ...process.env };
-    env.SENTRY_ANR_CHILD_PROCESS = 'true';
+    log(`Spawning worker with entryScript:'${options.entryScript}'`);
 
-    if (options.captureStackTrace) {
-      env.SENTRY_INSPECT_URL = startInspector();
-    }
-
-    log(`Spawning child process with execPath:'${process.execPath}' and entryScript:'${options.entryScript}'`);
-
-    const child = spawn(process.execPath, [options.entryScript], {
-      env,
-      stdio: logger.isEnabled() ? ['inherit', 'inherit', 'inherit', 'ipc'] : ['ignore', 'ignore', 'ignore', 'ipc'],
-    });
-    // The child process should not keep the main process alive
-    child.unref();
+    const inspectorUrl = options.captureStackTrace ? startInspector() : undefined;
+    const worker = new Worker(options.entryScript, { workerData: { inspectorUrl } });
+    // The worker should not keep the main process alive
+    worker.unref();
```

Comment:
`undef` is available after Node 15.4.0, but we support Node < 12

Reply:
The docs suggest `unref` has been supported since v10.5.0:
https://nodejs.org/api/worker_threads.html#workerunref 

The minimum supported Node version for ANR via a worker thread is v12.11.0 since before that `--experimental-worker` was required.

Reply:
Yeah, you're right. I think I was confused with BroadcastChannel.